sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -672,6 +672,24 @@ private bool TryGetAttributeProperties(AttributeData attributeData, Location? at
                 return true;
             }
 
+            private bool WriteArgumentToMetadata(TypedConstant? namedArgument)
```

Comment:
The method name suggests it is "writing" something. but this method do some checks and return a boolean value. Should we rename this method to make it more readable?